sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/consumer_client.hpp

```diff
@@ -29,9 +32,20 @@ namespace Azure { namespace Messaging { namespace EventHubs {
      */
     Azure::Core::Http::Policies::RetryOptions RetryOptions{};
 
-    /**@brief  Message sender options.
-     */
-    Azure::Core::Amqp::_internal::MessageReceiverOptions ReceiverOptions{};
+    /**@brief If enabled, generate verbose logging traces for this client and related classes.*/
+    bool VerboseLogging{false};
+
+    /** @brief Maximum message size for messages being sent. */
+    Azure::Nullable<std::uint64_t> MaxMessageSize;
+
+    /** @brief Name of the consumer client. */
+    std::string Name{};
```

Comment:
This field name is quite generic.
Consider `ConsumerName`

Reply:
It is generic, however this is `ConsumerClientOptions::Name` - the field `Name` is implicitly scoped to `ConsumerClient`. Renaming it `ConsumerClientName` means the field is `ConsumerClientOptions::ConsumerClientName` which is redundant.